packages/tailwindcss-language-service/src/diagnostics/getInvalidTailwindDirectiveDiagnostics.ts

```diff
@@ -98,3 +70,87 @@ export function getInvalidTailwindDirectiveDiagnostics(
 
   return diagnostics
 }
+
+function validateLayerName(
+  state: State,
+  layerName: string,
+): { message: string; suggestions: string[] } | null {
+  if (state.v4) {
+    // `@tailwind utilities` is valid
+    if (layerName === 'utilities') {
+      return null
+    }
+
+    // `@tailwind base | preflight` do not exist in v4
+    if (layerName === 'base' || layerName === 'preflight') {
+      return {
+        message: `'@tailwind ${layerName}' is no longer available in v4. Use '@import "tailwindcss/base"' instead.`,
+        suggestions: [],
+      }
+    }
+
+    // `@tailwind components | screens | variants` do not exist in v4
+    if (layerName === 'components' || layerName === 'screens' || layerName === 'variants') {
+      return {
+        message: `'@tailwind ${layerName}' is no longer available in v4. Use '@tailwind utilities' instead.`,
+        suggestions: [],
+      }
+    }
+
+    let parts = layerName.split(/\s+/)
+
+    // `@tailwind utilities source(…)` is valid
+    if (parts[0] === 'utilities' && parts[1]?.startsWith('source(')) {
+      return null
+    }
+
+    return {
+      message: `'${layerName}' is not a valid value.`,
+      suggestions: [],
+    }
+  }
+
+  let valid = ['utilities', 'components', 'screens']
+
+  if (semver.gte(state.version, '1.0.0-beta.1')) {
+    valid.push('base')
+  } else {
+    valid.push('preflight')
+  }
+
+  let hasVariantsDirective = state.jit && semver.gte(state.version, '2.1.99')
+
+  if (hasVariantsDirective) {
+    valid.push('variants')
+  }
+
+  if (valid.includes(layerName)) {
+    return null
+  }
+
+  let suggestable = valid
```

Comment:
```suggestion
  let suggestible = valid
```
(also occurs in other spots)

Reply:
TIL I've been spelling this wrong for… basically ever. wow

Reply:
Oh hey it was already like this so not my typo 🎉 still I didn't realize it was wrong lol

Reply:
Wait no is right…

> suggestable
> Capable of being suggested

suggestible is like "open to suggestion" or something. nothing to see here lol